spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -47,19 +47,163 @@ definitions:
      - discontinuity_counter:
             type: u8
             desc: |
-              Signal phase discontinuity counter. 
+              Signal phase discontinuity counter.
               Increased for every discontinuity in phase.
      - bias:
             type: s32
             units: 0.1 mm
             desc: Phase bias for specified signal
 
+ - SvId:
```

Comment:
Would we want to use GNSS signal here?

Reply:
We could reuse that; the reason I didn't is because band is not used or needed for this message, so I felt that including the band was a bit of a red herring.

Reply:
I meant to remove `SvId` and use `GNSSSignal` instead. If the signal is not applicable and just the satellite - is that something we've done before? If that's the case, maybe it belongs in `gnss.yaml` alongside `GNSSSignal`.

Reply:
Let's use the `SvId` type but move it into `gnss.yaml` (worth renaming `GNSSSat`?).

Reply:
done.